### ISSUE TITLE
The docs are not renderred correctly on http://docs.dit.io

### ISSUE THREAD

**Issue by ilongshan** [dit/dit#163]
**Describe the bug**
The ipython docs are not correctly rendered on http://docs.dit.io. Please help to fix it.

![Snip20190826_1](https://user-images.githubusercontent.com/7447401/63653670-81ee0580-c7a2-11e9-89ec-bef10ee4cf0e.png)

**Comment by Ejjaffe** [comment]
![image](https://user-images.githubusercontent.com/38086802/115980638-42218780-a55c-11eb-9b6d-46def47aebdc.png)
Can someone help me figure out where this IPYNB's located so I can make a pull request myself?

**Comment by Autoplectic** [comment]
@Ejjaffe that's not in an ipynb, unfortunately. When the documentation was originally written, we used the ipython sphinx directive, which lets you put the input code into a block, and during documentation compilation it executes that code and dynamically sticks the output into the docs, ensuring that things are up to date. Unfortunately, that sphinx extension hasn't seen any love in a long time, and seems to be faltering. I can try to look in to that later, but it's going to be a lower priority issue.

**Comment by Ejjaffe** [comment]
@Autoplectic that's a bummer! I think the string typo should be a quick fix. Also I'm not super familiar with sphinx but I would be happy to research the issue and maybe take a stab at it. Are you currently using [nbsphinx?](https://nbsphinx.readthedocs.io/en/0.8.3/)

**Comment by Autoplectic** [comment]
No, I don’t think so. We do have a few example notebooks, but most of the
documentation is hand-written in restructuredtext. We use several plugins
for things like generating bibliographies and other things.

________________________
Ryan G. James
Information Theorist
Complexity Sciences Center
University of California, Davis
